addons/website/views/snippets/snippets.xml

```diff
@@ -1557,9 +1557,7 @@
         <we-input string="Line height"
                   data-customize-website-variable="null"
                   data-variable="body-line-height"
-                  data-fake-unit="× "
-                  data-unit="rem"
-                  data-save-unit="rem"/>
```

Comment:
@bso-odoo Thanks for the fix :+1: Although, since 16.4 is already used by users, but was released recently, let's agree to do this:
- Keep your XML patch here, normally not ok since XML change in stable but let's say it is recent enough to justify this :)
- But let's also do it in JS: in the `_patchForComputeSnippetTemplates`, remove the unit if present on that element. That way existing users in 16.4 will also be able to set up line height correctly and save them in their CSS palette with rem unit too. Good news enough, we'll probably never have to care about this anymore after 17.0 since those definitions will be client side with the OWL convertion.
- As you said: "In case the value had been edited before this fix, it will be shown with the "rem" text within the option" -> let's say it is ok. Better that than showing them a fake unitless unit they cannot fix.
- As you did: let's not try to remove the rem unit when using the wrong user values. It would have been recent enough to ignore the potential change of visual on user websites... but it would have meant having to keep a crappy patch in SCSS forever since we cannot migrate those values properly for now.

So just one change to do: on top of what you did, also patch `_patchForComputeSnippetTemplates` :pray:

Reply:
@qsm-odoo Added stable patch in `_patchForComputeSnippetTemplates`.